README.md

```diff
@@ -72,6 +72,7 @@ $ kubectl get opsrc upstream-community-operators -o=custom-columns=NAME:.metadat
 NAME                           PACKAGES
 upstream-community-operators   federationv2,svcat,metering,etcd,prometheus,automationbroker,templateservicebroker,cluster-logging,jaeger,descheduler
 ```
+**_Note_**: Please do not install [upstream-community-operators](deploy/upstream/upstream.operatorsource.cr.yaml) and [community-operators](deploy/examples/community.operatorsource.cr.yaml) `OperatorSources` on the same cluster. The rule of thumb is to install [community-operators](deploy/examples/community.operatorsource.cr.yaml) on OpenShift clusters and [upstream-community-operators](deploy/upstream/upstream.operatorsource.cr.yaml) on upstream Kubernetes clusters.
```

Comment:
The upstream operator source is in `deploy/examples/upstream.operatorsource.cr.yaml`

Reply:
It has been moved to deploy/upstream as part of this PR as per @kevinrizza's request.